### ISSUE TITLE
I Need Help!

### ISSUE THREAD

**Issue by daltonjames777** [karim0sec/Bulk-Tiktoks-Downloader#31]
I need help. The script has worked very well so far, the only problem I have is that it saves from 3 to a maximum of 30 videos, regardless of how many videos the TikTok account has, writes descriptions saved, and starts downloading videos without saving the entire list, TikTok has made some changes as far as I understand! Thank You

**Comment by karim0sec** [comment]
yeah tiktok showing captcha and fight bots now i will retire that repo
